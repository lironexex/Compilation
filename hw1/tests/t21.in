// comments 2 - escape sequence inside comments (of both types) have no effect

// for example, this escape sequence: \u{41} is just ignored
// \u{00000069} \q \U{} \u{} \" \\ \n \r \t 
/* \u{00000069} \q \U{} \u{}
 \"
  \\
   \n
    \r
     \t
*/ 
// "this is not technically a string! \u{41} \\ \t \r \n this is all a part of the comment" 
/* "this is not technically a string! \u{41} \\ \t \r \n this is all a part of the comment" */ 
