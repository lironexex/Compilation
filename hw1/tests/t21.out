1 COMMENT 1
3 COMMENT 1
4 COMMENT 1
11 COMMENT 7
12 COMMENT 1
13 COMMENT 1
